mne/surface.py

```diff
@@ -1689,12 +1691,107 @@ def marching_cubes(image, level):
     mc.Update()
     polydata = mc.GetOutput()
 
+    # get verts and triangles
     verts = numpy_support.vtk_to_numpy(polydata.GetPoints().GetData())
     triangles = numpy_support.vtk_to_numpy(
         polydata.GetPolys().GetConnectivityArray()).reshape(-1, 3)
     verts = np.flip(verts, axis=1)
     triangles = np.flip(triangles, axis=1)
-    return verts, triangles
+
+    # compute curvature and get it
+    curvature = vtkCurvatures()
+    curvature.SetCurvatureTypeToGaussian()
+    curvature.SetInputData(polydata)
+    curvature.Update()
+    curv_data = curvature.GetOutput()
+    curv = numpy_support.vtk_to_numpy(
+        curv_data.GetPointData().GetArray("Gauss_Curvature"))
```

Comment:
As discussed on Discord, I don't think we want/need curvatures for volume ROIs as they likely aren't as meaningful as they are for cortex (where the sulcal/gyral distinction is important)